internal/pkg/describe/env_test.go

```diff
@@ -4,24 +4,29 @@
 package describe
 
 import (
+	"errors"
+	"fmt"
+	"testing"
+
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/deploy/cloudformation/stack"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/describe/mocks"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
-	"testing"
 )
 
 type envDescriberMocks struct {
-	mockStoreService   *mocks.MockstoreSvc
+	mockStoreService         *mocks.MockstoreSvc
```

Comment:
Do we actually need the `mockStoreService` for these tests? It looks like we don't call anything on the store for the `Describe()` method, so I would probably leave this out for now (which means you don't need it in lines 99 and 110)

I'm not sure if you'll need to mock the store for other things you might want to test in the describer -- I do like to have the `fooMocks` struct generally mirror the `Foo` thing you're mocking (in this case, "foo" == `EnvDescriber`) so that instantiating it in the test is as close to the actual constructor as possible, but if it's not used in a particular set of unit tests I might leave it out.

Reply:
I had taken this out yesterday, on your advice, but have since put it back in because I am now back to writing tests for Describe, as I have changed FilterAppsForEnv to filterAppsForEnv.